lib/util/object-to-string.js

```diff
@@ -12,10 +12,13 @@ import { flatten } from 'ramda';
  * returns: `SIV_ATTRIBUTE(id)==[123]`
  */
 export default function convertObjectToString(filter) {
-  if (filter.constructor === Array) {
+  if (filter === null) {
```

Comment:
```suggestion
  if (!filter) {
```
I might be wrong but this way we won't have to check if filter exists in the following instructions on lines 18/21?

Reply:
This is an interesting thing. Maybe I am overthinking it but I would say that if the function is called `convertObjectToString` it should convert an object to a string. As I am seeing it, `null` is quite a special case, that's why I did it this way. If the `filter` is `undefined`, `0`, or `''`, it would just return an empty string as the name suggests.

What do you think @shamrt @Yathi ?

Reply:
If we want a single return value for these falsy inputs, I would probably just use empty string rather than `null`.

Reply:
That's a fair point. Do we want to handle undefined as well? If so, the library uses Ramda, so we could go
```suggestion
  if (isNil(filter)) {
```

Reply:
Is there a test for handling `undefined`?

Reply:
There is not. The tests cover more real world scenarios.

Reply:
I'll add some tests for `convertObjectToString`.

Reply:
Okay, I'm catching up to you now, and have actually read the function. Its name, `convertObjectToString`, made me think it did something that it doesn't. Having now fully reviewed the implementation, I can see that you're right: it would return `''` if given `undefined`.

I don't know whether we can claim for certain that `filter` will never be `undefined` in the real world — especially if this function gets used in a place where it's more-directly exposed to the end-user — so I think we should augment the test module with all possible conditions. That would not only cover those scenarios, but also serve as a form of documentation as well as help during future refactorings.